Less/footer.less

```diff
@@ -0,0 +1,75 @@
+.sign-up {
+  color: @white;
+  // margin-top: 35rem;
+  margin-bottom: 5rem;
+  padding: 10rem;
+
+  .flex-content(space-between, center);
+  position: relative;
```

Comment:
nice use of paramteric mixins here and throughout your LESS files --looks great so far